java-logging-appinsights/README.md

```diff
@@ -70,10 +70,9 @@ TelemetryClient can be `autowired`  to implement custom telemetry metrics.
 
 #### Provide Instrumentation Key
      
-Add below to your application properties to be compatible with current shared CNP configurations.
-
+Set environment variable: AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY, if you are deploying using the CNP pipeline this will be automatically added for you
```

Comment:
```suggestion
Set the environment variable: AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY, if you are deploying using the CNP pipeline this will be automatically added for you
```

also:
```
You can also set it using a spring property (useful for tests):
```